src/lib.rs

```diff
@@ -504,7 +512,7 @@ impl Renderer {
         scene: &Scene,
         surface: &SurfaceTexture,
         params: &RenderParams,
-    ) -> Result<Option<BumpAllocators>, AsyncRenderError> {
+    ) -> VResult<Option<BumpAllocators>> {
```

Comment:
I think you can just use `Result` instead of `VResult`. This is more common IME.

Reply:
So should I make them all explicit?

Reply:
No I meant you could just do:

```rust
pub(crate) type Result<T> = Result<T, VelloError>;
```

Reply:
I wouldn't do that. For a `Box<dyn Error>` this is ok, but for specific Errors, it should at least have the `V` before.

Reply:
Well I guess you could do that everytime you're only using `Result<T, VelloError>` in a module (and no other type of `Result`, it's only syntax sugar to reduce boilerplate anyway, and that is relatively common practice IME. It should of course not be exported from the lib (in case I certainly would agree).

Reply:
Nevermind, won't happen. In the same file i also use `Result<(), wgpu::Error>` so it gives name clash.

Reply:
I believe the fix for that is:
```rust
type Result<T, E=VelloError> = Result<T, E>;
```